Userland/Libraries/LibCore/StandardPaths.cpp

```diff
@@ -51,9 +52,16 @@ DeprecatedString StandardPaths::downloads_directory()
 
 DeprecatedString StandardPaths::config_directory()
 {
+    if (auto* config_directory = getenv("XDG_CONFIG_HOME"))
+        return LexicalPath::canonicalized_path(config_directory);
```

Comment:
if not serenity?

Reply:
I don't really have an opinion, if someone sets `XDG_*` on Serenity, it might be friendly to respect it :man_shrugging: